### ISSUE TITLE
UI: Data not refreshed on job completion

### ISSUE THREAD

**Issue by Pearl1594** [apache/cloudstack#5279]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
main
~~~

##### SUMMARY

Performing operations like disabling of hosts, host removal, offering deletion, etc, does not refresh the page i.e., change in state / the effect of the operation isn't observed until one manually refreshes the page

##### STEPS TO REPRODUCE

~~~
Attempt any of the operations above - say, disable a host; while the operation is successful, the form doesn't close and the host doesn't appear to be in the disabled state until one manually refreshes the page
~~~

##### EXPECTED RESULTS

~~~
Form closes on successful execution of the operation and the state change is immediately reflected
~~~

##### ACTUAL RESULTS

~~~
Page data not refreshed and form/dialog-box does not close on successful completion of operation
~~~

**Comment by nvazquez** [comment]
@davidjumani can you please work on this?

**Comment by shwstppr** [comment]
@Pearl1594 can you please check latest main now. I feel this should be fixed now with #5285 cc @nvazquez
![actioncheck](https://user-images.githubusercontent.com/153340/128562079-665c22da-58f8-4d4d-939a-7fb5cbee3e82.gif)

**Comment by davidjumani** [comment]
Tested, fixed with #5285 . @Pearl1594 Can you check and update / close

**Comment by Pearl1594** [comment]
Thanks, Seems to be fixed. Closing the issue.
